### PR TITLE
Week 6 planning

### PR DESCRIPTION
**Pull request by m4v15**

### CODE REVIEW

File: coursebook/week-6/README.md [hunk 1]
```diff
@@ -19,7 +19,11 @@ and work through in pairs if time remaining (help each other first)!
 12:45pm - 2.45pm:
 [workshop on SQL commands and psql](https://github.com/foundersandcoders/postgres-workshop).
 
+<<<<<<< HEAD
```
Comment (m4v15) [inline coursebook/week-6/README.md]: merge conflict?

File: coursebook/week-6/README.md [hunk 2]
```diff
@@ -34,16 +38,21 @@ and work through in pairs if time remaining (help each other first)!
...
+2:00pm - 4:00pm -- [Research Afternoon](./research-afternoon.md) -- Review and
+Present research topics
 
 #### Day 3
 
 8:30am - 9:30am:
 [morning challenge](https://github.com/foundersandcoders/db-morning-challenge)
 
-9:30am onwards: projects
+9:30am - 10:30am Design Burst
```
Comment (m4v15) [inline coursebook/week-6/README.md]: I think leave this in for now, but maybe this won't happen @jema28 ?

File: coursebook/week-6/README.md (hunk 1 above)
Comment (ghassanmas) [reply coursebook/week-6/README.md]: that is already removed

Comment (m4v15) [reply coursebook/week-6/README.md]: yeah sorry not sure what's going on

File: coursebook/week-6/README.md (hunk 2 above)
Comment (jema28) [reply coursebook/week-6/README.md]: Yeh, it definitely won't take  1 hour, 20 - 30 min tops. @ghassanmas It'll be a design resources session not a design burst 🐢

Comment (m4v15) [reply coursebook/week-6/README.md]: I changed this to 9:30-10:00, I think we can merge now?

**Pull request merged**